frontend/src/components/accounts/AccountDetails.tsx

```diff
@@ -46,16 +49,23 @@ export default ({ account }: Props) => {
             text={<Balance amount={account.locked} />}
           />
         </Col>
-        <Col md="4">
+        <Col md="3">
           <CardCell
-            title="Storage (Used / Paid)"
+            title="Storage Used (Bytes)"
+            imgLink="/static/images/icon-storage.svg"
+            text={`${account.storageUsage.toLocaleString()} B`}
+          />
+        </Col>
+        <Col md="2">
+          <CardCell
+            title="Last Paid"
             imgLink="/static/images/icon-m-block.svg"
-            text={`${account.storageUsage.toLocaleString()} / ${account.storagePaidAt.toLocaleString()}`}
+            text={`#${account.storagePaidAt.toLocaleString()}`}
```

Comment:
Let's have `storagePaidAtBlockHash` in the state and fetch the block info (on `componentDidMount` / `componentDidUpdate`) via `BlocksApi` by `account.storagePaidAt` block height to receive the block hash. Here (in the `render` method) we should just show:

* a plain `#${account.storagePaidAt.toLocaleString()}` (when `state.storagePaidAtBlockHash` is not available)
* or
    ```jsx
    <BlockLink blockHash={this.state.storagePaidAtBlockHash}>
      {`#${account.storagePaidAt.toLocaleString()}`}
    </BlockLink>
    ```

Reply:
@frol if we do so, the test for AccountDetails will not okay since we have BlocksApi here to load.

Reply:
@icerove Does the test suite trigger `componentDidMount` or `componentDidUpdate`? Can we use the same workaround as we have in `Transactions.tsx` (define the Api as null and assign it on the first use)?

Reply:
will have try

Reply:
sorry, but not able to keep test with Api use

Reply:
I think the error happen in import not in component 
<img width="572" alt="image" src="https://user-images.githubusercontent.com/33007476/75498654-9c38b400-597c-11ea-8675-20aeb4e05a47.png">

Reply:
Both import and tests work fine for https://github.com/nearprotocol/near-explorer/blob/321ef470f12c48527cc171f0f4640ae09a366646/frontend/src/components/transactions/Transactions.tsx. Commit the change and let's have a look.

Reply:
@frol I do not think we have a test for Transactions.tsx. where can I see that and learn?

Reply:
Oh well, you are right. Then, we have two options:

1. Make `frontend/src/libraries/config.ts` non-failing (add some checks around `getConfig()` and set some empty values if it is `undefined`)
2. Mock it with jest (`frontend/jest.setup.js` already has some mocks)